### PR TITLE
Broker storage

### PR DESCRIPTION
**Pull request by chemicL**

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -21,7 +21,7 @@ object KafkaUtilsBuild extends Build {
     libraryDependencies ++= Seq(
       "log4j" % "log4j" % "1.2.17",
       "org.scalatest" %% "scalatest" % "1.9.1" % "test",
-      "org.apache.kafka" %% "kafka" % "0.8.1"))
+      "org.apache.kafka" %% "kafka" % "0.8.2.1"))
```
Comment (pawelszymczyk) [inline project/Build.scala]: test

**Comment by adamdubiel** [comment]
some change in readme would be welcome ;)

File: README.md [hunk 2]
```diff
@@ -40,6 +40,7 @@ java -cp KafkaOffsetMonitor-assembly-0.2.1.jar \
      --port 8080 \
      --refresh 10.seconds \
      --retain 2.days
+     --brokerStorage true
```
Comment (chemicL) [inline README.md]: I needed to explicitly provide "true", as the args parsing doesn't support empty flags https://github.com/quantifind/Sumac/issues/17

**Comment by pszymczyk** [comment]
:+1:

**Pull request closed without merging**